model/src/voice/voice_state.rs

```diff
@@ -1,12 +1,16 @@
 use crate::{
-    guild::Member,
+    guild::member::{Member, OptionalMemberDeserializer},
     id::{ChannelId, GuildId, UserId},
 };
-use serde::{Deserialize, Serialize};
+use serde::{
+    de::{DeserializeSeed, Deserializer, Error as DeError},
+    Deserialize, Serialize,
+};
 use serde_mappable_seq::Key;
+use serde_value::Value;
```

Comment:
`serde_value::Value` should be avoided due to being slow to (de)serialize into/from. An example is the [reaction optimisation PR](https://github.com/twilight-rs/twilight/pull/265) (#265), which optimised the deserialisation time of a reaction by 47%, or the [typing start PR](https://github.com/twilight-rs/twilight/pull/261) (#261) which optimised deserialisation time of a Typing Start event by 55%. Each of these were done just by removing a value.